.github/workflows/tests.yml

```diff
@@ -58,10 +58,28 @@ jobs:
         run: |
           go mod download
 
+      - name: Make LXD tarball and unpack it
+        env:
+          CUSTOM_VERSION: "test"
+        run: |
+          make dist
+          tar -xzf lxd-test.tar.gz
+
+      - name: Build LXD dependencies
+        run: |
+          cd lxd-test
+          make deps
+
       - name: Run LXD build
         run: |
+          cd lxd-test
           make
 
+      - name: Cleanup generated files
+        run: |
+          rm -rf lxd-test
+          sudo rm -f /tmp/lxddoc.log
```

Comment:
I am not sure why this file is problematic, but if `/tmp/lxddoc.log` is not removed prior running `go test ./...`, a permission issue occurs during the test.

Similarly, if I move the unit tests before the tarball is built, the same issue occurs during `make dist`.

So, removing it prior unit tests seems to work, but I just wanted to emphasize this.

Reply:
@gabrielmougard what is lxddoc.log and why is it writing into /tmp always? Why isn't lxddoc using stdout/stderr for its logging?

Reply:
@tomponline this willl be fixed here https://github.com/canonical/lxd/pull/12101